recipes/issho/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "issho" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 430394f679ca5bedf603c4f68ec576149a90de1f70bbb82af2e50fa8c634bf06 
+
+build:
+  skip: true  # [py<34]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
```

If it is made `noarch` you can constrain the Python version with:
```suggestion
    - python >=3.5
```